test/integration/full-content/fixtures/core__column.json

```diff
@@ -23,9 +50,36 @@
                 "name": "core/paragraph",
                 "isValid": true,
                 "attributes": {
-                    "content": [
-                        "Column One, Paragraph Two"
-                    ],
+                    "content": {
+                        "formats": [
```

Comment:
I’m a bit concerned about the number of entries this will create in memory.
Some people will paste a book into Gutenberg …
Those array can become huuuuuge

@mcsf - do we have an example post which is a book like to test it with? I know that you have some heavy stuff to test the parser itself.

Reply:
Performance is also a concern of mine.

> Some people will paste a book into Gutenberg […] do we have an example post which is a book like to test it with?

Yes: in fact, we have two _actual_ books, _Moby Dick_ and _Pygmalian_, here:

https://github.com/dmsnell/gutenberg-document-library/tree/master/library

Reply:
Replied to you in chat, but I'll reply here soon about performance.